cmds/connect.go

```diff
@@ -28,7 +28,11 @@ func Connect() *cli.Command {
 			},
 		},
 		Action: func(cCtx *cli.Context) error {
-			session := strings.Trim(cCtx.Args().First(), "\"'")
+			// Some shells sends the as the session name the icon and the session name in different values
+			// This means that getting the first value might not be always correct
+			// sesh connect  session_name
+			// sesh connect " session_name"
+			session := strings.Trim(cCtx.Args().Get(cCtx.Args().Len()-1), "\"'")
```

Comment:
What do you think of changing this to just concatenate all arguments into a string? So something all of these examples would work:
```sh
sesh connect tmux config # becomes "tmux config"
sesh connect  dotfiles # becomes " dotfiles"
sesh connect anything you want it to be # becomes "anything you want it to be"
```